Terminal.Gui/Views/TabView.cs

```diff
@@ -187,9 +187,12 @@ public override void OnDrawContent (Rect contentArea)
 				// How much space do we need to leave at the bottom to show the tabs
 				int spaceAtBottom = Math.Max (0, GetTabHeight (false) - 1);
 				int startAtY = Math.Max (0, GetTabHeight (true) - 1);
+				var lc = new LineCanvas () { Lines = new List<StraightLine> (LineCanvas.Lines) };
 
```

Comment:
This looks hacky.

The issue is that `DrawFrame` modifies `LineCanvas` and we want to prevent that behaviour?  If this is the case, then adding a `bool` parameter to `DrawFrame` allowing use a fresh `LineCanvas` would be better approach.

```csharp
/// <summary>
/// Draw a frame based on the passed bounds to the screen relative.
/// </summary>
/// <param name="bounds">The bounds view relative.</param>
/// <param name="lineStyle">The line style.</param>
/// <param name="attribute">The color to use.</param>
/// <param name="mergeWithLineCanvas">When drawing the frame, allow it to integrate (join) to other frames in other controls.
/// Or false to simply draw the rect exactly with no side effects.</param>
public void DrawFrame (Rect bounds, LineStyle lineStyle, Attribute? attribute = null, bool mergeWithLineCanvas=true)
```